script.js

```diff
@@ -0,0 +1,524 @@
+        eng: {upperCase: 'B', lowerCase: 'b'},
+        keyCode: 'KeyB'
+    }, {
+        rus: {upperCase: 'Т', lowerCase: 'т'},
+        eng: {upperCase: 'N', lowerCase: 'n'},
+        keyCode: 'KeyN'
+    }, {
+        rus: {upperCase: 'Ь', lowerCase: 'ь'},
+        eng: {upperCase: 'M', lowerCase: 'm'},
+        keyCode: 'KeyM'
+    }, {
+        rus: {upperCase: 'Б', lowerCase: 'б'},
+        eng: {upperCase: '<', lowerCase: ','},
+        keyCode: 'Comma'
+    }, {
+        rus: {upperCase: 'Ю', lowerCase: 'ю'},
+        eng: {upperCase: '>', lowerCase: '.'},
+        keyCode: 'Period'
+    }, {
+        rus: {upperCase: ',', lowerCase: '.'},
+        eng: {upperCase: '?', lowerCase: '/'},
+        keyCode: 'Slash'
+    }, {
+        rus: {upperCase: '▲', lowerCase: '▲'},
+        eng: {upperCase: '▲', lowerCase: '▲'},
+        keyCode: 'ArrowUp'
+    }, {
+        rus: {upperCase: 'Shift', lowerCase: 'Shift'},
+        eng: {upperCase: 'Shift', lowerCase: 'Shift'},
+        keyCode: 'ShiftRight'
+    },],
+    [{
+        rus: {upperCase: 'Ctrl', lowerCase: 'Ctrl'},
+        eng: {upperCase: 'Ctrl', lowerCase: 'Ctrl'},
+        keyCode: 'ControlLeft'
+    }, {
+        rus: {upperCase: 'Ru', lowerCase: 'Ru'},
+        eng: {upperCase: 'En', lowerCase: 'En'},
+        keyCode: 'langKey'
+    }, {
+        rus: {upperCase: 'Alt', lowerCase: 'Alt'},
+        eng: {upperCase: 'Alt', lowerCase: 'Alt'},
+        keyCode: 'AltLeft'
+    }, {
+        rus: {upperCase: '', lowerCase: ''},
+        eng: {upperCase: '', lowerCase: ''},
+        keyCode: 'Space'
+    }, {
+        rus: {upperCase: 'Alt', lowerCase: 'Alt'},
+        eng: {upperCase: 'Alt', lowerCase: 'Alt'},
+        keyCode: 'AltRight'
+    }, {
+        rus: {upperCase: '◄', lowerCase: '◄'},
+        eng: {upperCase: '◄', lowerCase: '◄'},
+        keyCode: 'ArrowLeft'
+    }, {
+        rus: {upperCase: '▼', lowerCase: '▼'},
+        eng: {upperCase: '▼', lowerCase: '▼'},
+        keyCode: 'ArrowDown'
+    }, {
+        rus: {upperCase: '►', lowerCase: '►'},
+        eng: {upperCase: '►', lowerCase: '►'},
+        keyCode: 'ArrowRight'
+    }, {
+        rus: {upperCase: 'Ctrl', lowerCase: 'Ctrl'},
+        eng: {upperCase: 'Ctrl', lowerCase: 'Ctrl'},
+        keyCode: 'ControlRight'
+    }],
+];
+
+//Create keyboard
+const createKeys = () => {
+    const fragment = document.createDocumentFragment();
+    keys.forEach(row => {
+        // Создаем отдельную строку из клавиш
+        const rowElement = document.createElement('div');
+        rowElement.classList.add('keyboard__row');
+        row.forEach(key => {
+            // Создаем отдельную клавишу в строке
+            const keyElement = document.createElement('div');
+            keyElement.classList.add('keyboard__key', key.keyCode);
+            if (key.keyCode === 'Tab' || key.keyCode === 'ControlLeft' || key.keyCode === 'AltLeft' || key.keyCode === 'AltRight' || key.keyCode === 'ControlRight') {
```

Comment:
Move `key.keyCode === 'Tab' || key.keyCode === 'ControlLeft' || key.keyCode === 'AltLeft' || key.keyCode === 'AltRight' || key.keyCode === 'ControlRight'` in a separate function and return true or false. 
if(isTabOrControlOrAltKey())